src/SFA.DAS.EmployerFinance.Messages/Events/StartedProcessingLevyDeclarationsEvent.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace SFA.DAS.EmployerFinance.Messages.Events
+{
+    public class StartedProcessingLevyDeclarationsEvent
+    {
+        public int SagaId { get; }
+        public DateTime PayrollPeriod { get; }
+        public DateTime Started { get; }
```

Comment:
I can't see any obvious usage of this property (or at least Rider can't 🙂). Is it used, or do we plan on using it, e.g. for logging or profiling? If not, can we remove it?

The same applies to `Started` in `StartedProcessingLevyDeclarationsAdHocEvent`.

Reply:
Yeah I included it in case there's any consumers that will be interested in the event, think it's redundant?

Reply:
Consumers could use the `TimeSent` message header, which should be basically equivalent.

Reply:
As discussed offline it can be useful to have the started date as a property on the event rather than querying it from the infrastructure.